portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,173 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script>$('.carousel').carousel({
+      interval: 20
+    })</script>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+  <body onload="addRandomQuote()">
+    <div id="nav-placeholder"></div>
+
+    <nav class="navbar  fixed-top navbar-expand-lg navbar-dark bg-dark">
+      <a class="navbar-brand" href="#">Jose's Portfolio</a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarSupportedContent">
+        <ul class="navbar-nav mr-auto">
+          <li class="nav-item active">
+            <a class="nav-link" href="#hobbies">Hobbies</a>
+          </li>
+          <li class="nav-item active">
+            <a class="nav-link" href="#code">Code</a>
+          </li>
+          <li class="nav-item active">
+            <a class="nav-link" href="#places">Places</a>
+          </li>
+      </div>
+    </nav>
+
+    <br>
+    <br>
+
+    <div class="jumbotron text-center container-fluied">
+      <h2 >Here is a quote that I like:</h2>
+      <blockquote class="blockquote">
+        <p id ="quote-container" class="mb-0 "></p>
+        <footer id="name-container" class="blockquote-footer"><cite title="Source Title"></cite></footer>
+      </blockquote>
     </div>
+    <div id="hobbies" class="section1 grad1 container-fluid">
+    <h1 style="color:#000000">Hobbies</h1>
+
+    <div class="card-group">
+      <div class="card">
+        <img src="images/food.jpg" class="cropped card-img-top" alt="...">
+        <div class="card-body">
+          <h5 class="section-in">Eating Healthy</h5>
+          <p class="card-text section-in">Lately, eating healthier has been one of my main priorities. Eating vegan food more frequently is definitedly something to get used to!</p>
+        </div>
+      </div>
+
+      <div class="card">
+        <img src="images/music.jpg" class="cropped card-img-top" alt="...">
+        <div class="card-body">
+          <h5 class="card-title section-in">Music</h5>
+          <p class="card-text section-in">It helps me get relaxed, inspired, and curious! I have explored into all types of genres and every day I find something different to listen to</small></p>
```

Comment:
Some of these lines are very long (mainly affecting readability in the the code review).
You can add line breaks here as much as you want -- it won't affect the rendering but will make the sourcecode more readable.

(Again, good editors will automatically line-wrap text for you at a configured length, often 80 to 120 characters)